charts/operator/rbac.go

```diff
@@ -0,0 +1,797 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +gotohelm:filename=_rbac.go.tpl
+package operator
+
+import (
+	"github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func ClusterRole(dot *helmette.Dot) []rbacv1.ClusterRole {
```

Comment:
Note for the future (not this PR). We could add the ability to load files into gotohelm and just read these from disk so they don't get out of sync. Similar to what I did here: https://github.com/redpanda-data/helm-charts/pull/1285